client/src/components/kanbancomponents/KanbanDetailedInfo.js

```diff
@@ -0,0 +1,63 @@
+
+import {
+    Switch,
+    Route,
+    NavLink,
+    useParams,
+    useRouteMatch
+} from "react-router-dom";
+import styled from "styled-components";
+
+function Characters() {
+    let { url, path } = useRouteMatch();
+
+    return (
+        <div>
+            <TopicInStyle>About Kanban</TopicInStyle>
+            <nav>
+                <NavLinkStyled to={url + "/KanbanBoard"}>The Kanban board</NavLinkStyled>
+                <NavLinkStyled to={url + "/History"}>History</NavLinkStyled>
+                <NavLinkStyled to={url + "/ForMe"}>what can it do for me?</NavLinkStyled>
+            </nav>
+
+            <Switch>
+                <Route exact path={path}>
+                    <TopicInStyle>Please select a topic about Kanban of which you would like further Info</TopicInStyle>
+                </Route>
+                <Route path={path + "/:characterName"}>
+                    <Character />
+                </Route>
+            </Switch>
+        </div>
+    );
+
+    function Character() {
+        const { characterName } = useParams();
+        return (
+            <div>
+                <TopicInStyle>{characterName}</TopicInStyle>
+            </div>
+        );
+    }
+}
+
+export default Characters;
+
+const NavLinkStyled = styled(NavLink)`
+  background: var(--primarypink);
+  color: var(--primaryblue);
+  border-radius: 30rem;
+  margin: 1rem;
+  padding: 1rem;
+
+  &:hover {
+    background: rebeccapurple;}
+
+    &.active{ width: 2rem;
+    };
```

Comment:
I think the `.active` class is now only used when you hover over an element. Was that intended?